packages/ceramic-http-client/src/document.ts

```diff
@@ -109,6 +103,14 @@ class Document extends EventEmitter {
     return true
   }
 
+  async pinDocument (): Promise<any> {
+    return await fetchJson(this._apiUrl + '/pin/add' + this.id)
+  }
+
+  async unpinDocument (): Promise<any> {
+    return await fetchJson(this._apiUrl + '/pin/rm' + this.id)
+  }
+
```

Comment:
Why are these here? We don't have any similar methods in the document class of ceramic-core.

Reply:
Got it. Removed it :)